### PR TITLE
Change SASS property-sort-order to concentric

### PR DESCRIPTION
**Pull request by RickvdStaaij**
Sorting sass on alphabetical order was not helping us at all, where concentric ordering does give us significant benefits.
https://rhodesmill.org/brandon/2011/concentric-css/

### CODE REVIEW

File: lib/codeStyles/Sass/sasslint/.sass-lint.yaml [hunk 1]
```diff
@@ -46,7 +46,9 @@ rules:
     no-trailing-whitespace: 2
     no-trailing-zero: 1
     one-declaration-per-line: 2
-    property-sort-order: 1
+    property-sort-order:
+        - 2
```
Comment (RickvdStaaij) [inline lib/codeStyles/Sass/sasslint/.sass-lint.yaml]: Should this still be warning? Or just error for us.

**Comment by RickvdStaaij** [comment]
@hjvvoorthuijsen please check the above

**Comment by sean-ww** [comment]
It’s an interesting idea 🤔
Doesn’t it over-complicate something that’s pretty simple? We rarely have a long list of properties to warrant it in my opinion. I also think it then makes our codebase harder to understand for those unfamiliar with the system. Is it worth learning? Even if you auto fix it, for the readability benefit it needs to be second nature that you know where to look. Does it work with auto fixing? And if so, are there still cases where it breaks (as with the alpha way)?
Maybe it takes getting used to. I’m curious what others think...

File: lib/codeStyles/Sass/sasslint/.sass-lint.yaml (hunk 1 above)
Comment (sean-ww) [reply lib/codeStyles/Sass/sasslint/.sass-lint.yaml]: I think that originally related to it being buggy, so it wouldn’t cause errors unnecessarily. It may no longer be the case, not sure.

**Comment by RickvdStaaij** [comment]
I've been ordering the css by the properties for a while now in my personal projects, and it makes it a lot easier to read and to find properties. Where with the alphabetical ordering, I'm confused all the time where properties go and I'm ding the ABC's over and over again in my head (yes, I can't :innocent:).

The concentric ordering makes a lot of sense and really helps with writing cleaner and more structured sass. Instead of having to do everything manually (which I am already doing anyway), this actually makes it easier if you forget where something should go.

On the `make fix` part. I introduced [husky](https://www.npmjs.com/package/husky) in combination with [lint-staged](https://github.com/okonet/lint-staged). This combo will fix and add all fixes to your commit upon committing. Really easy and works like a charm. Only beautiful commits are then added to your project.

So, long story short, I'm a big fan of this ordering.
